ntbs-service/Services/AuthorizationService.cs

```diff
@@ -187,25 +187,17 @@ private bool UserBelongsToResidencePhecOfNotification(Notification notification)
                 _userPermissionsFilter = await GetUserPermissionsFilterAsync(user);
             }
 
-            if (_userPermissionsFilter.Type == UserType.NhsUser)
-            {
-                notifications = notifications.Where(n => _userPermissionsFilter.IncludedTBServiceCodes.Contains(n.HospitalDetails.TBServiceCode));
-            }
-            else if (_userPermissionsFilter.Type == UserType.PheUser)
-            {
+            notifications = notifications.Where(n =>

```

Comment:
For national team users you've now made this query less efficient, as you've added a bunch of where clauses that we know aren't actually going to exclude anything

Reply:
Added a ternary operator here